### ISSUE TITLE
How certain can a core-top age be?

### ISSUE THREAD

**Issue by andydawson** [PalEON-Project/stepps-baconizing#11]
Right now if the error is 0 for the core top we reset it to be the integer that is the ceiling of the mean of the radiocarbon errors. But for some sites it appears to be 1, or something small. Should we reset those, or are we OK with being that certain?

**Comment by SimonGoring** [comment]
No, core tops, when they are modern should be 1 at most since we know the
date of sampling.  Right?  Only when the core top isn't modern should it be
the mean, and even then I'd suggest it should be more likely equivalent to
the next youngest date.

**Comment by andydawson** [comment]
Well I agree that we know with certainty the date of the actual core top, but thought that during sampling there was some mixing that happened in the upper-most sediment that led to some uncertainty. I found a previous email thread where @IceAgeEcologist suggested setting the core top errors to 5 due to this issue. 

Perhaps this mixing during sampling doesn't really matter for the age model though, we only care about the age at the depth, and not what is actually in the sediment, right? Or is it the depth measurement as well from the upper-most sediment that gets affected when you sample?

I can't recall why it was implemented the way I was doing it, so good thing we are having this convo on the record ;) . 

Until I hear anything further, I'm setting it to 5 when it is 0.

Ok, and finally, how do we define modern in this case? Anything that is after 1950?
